### PR TITLE
move es pillows to kafka only

### PR DESCRIPTION
**Pull request by emord**
Opening this up for tests. It'll move all pillows to kafka checkpoints

### CODE REVIEW

File: corehq/ex-submodules/pillowtop/checkpoints/manager.py [hunk 1]
```diff
@@ -150,6 +150,119 @@ def fire_change_processed(self, change, context):
...
+                        topic=topic_partition[0],
+                        partition=topic_partition[1],
+                        defaults={'offset': offset}
+                    )
+
+    def touch(self, min_interval):
+        return False
+
+
+class WrappedCheckpoint(object):
```
Comment (dimagimon) [inline corehq/ex-submodules/pillowtop/checkpoints/manager.py]: F811 redefinition of unused 'WrappedCheckpoint' from line 153

File: corehq/ex-submodules/pillowtop/checkpoints/manager.py [hunk 2]
```diff
@@ -150,6 +150,119 @@ def fire_change_processed(self, change, context):
...
+    def __init__(self, kafka_seq):
+        self.kafka_seq = kafka_seq
+
+    @property
+    def wrapped_sequence(self):
+        return self.kafka_seq
+
+
+class KafkaPillowCheckpoint(PillowCheckpoint):
```
Comment (dimagimon) [inline corehq/ex-submodules/pillowtop/checkpoints/manager.py]: F811 redefinition of unused 'KafkaPillowCheckpoint' from line 164

**Comment by emord** [comment]
@snopoke I changed the management command to only use kafka checkpoint here https://github.com/dimagi/commcare-hq/pull/16473#discussion_r120386793

We still need to use django pillow checkpoint until this is merged, which i planned on doing monday once I get off interrupt.

**Review by snopoke — APPROVED** [review]

**Comment by snopoke** [comment]
not merging due to label

**Comment by emord** [comment]
found a bug today that affects pillows with multiple topics. Will fix and merge on monday

**Comment by emord** [comment]
Added partitioning to the case pillow as I think that's all we need for right now.

File: corehq/apps/change_feed/consumer/feed.py [hunk 3]
```diff
@@ -80,9 +80,6 @@ def iter_changes(self, since, forever):
             offsets = [
                 copy(self._processed_topic_offsets)
             ]
-            topics_missing = set(self._topics) - checkpoint_topics
-            for topic in topics_missing:
-                offsets.append(topic)  # consume all available partitions
```
Comment (snopoke) [inline corehq/apps/change_feed/consumer/feed.py]: what's the reason for removing this?

Comment (emord) [inline corehq/apps/change_feed/consumer/feed.py]: documented in the commit message https://github.com/dimagi/commcare-hq/pull/16020/commits/2107e8ee5a7319712d77edc2dd6cd45ded751d0d

**Comment by emord** [comment]
:white_check_mark:  @snopoke I think this is ready to go today provided I answered your question well

**Pull request merged**